StoreSpidLogs/__test__/index.test.ts

```diff
@@ -3,15 +3,32 @@
 
 process.env = {
   APPINSIGHTS_INSTRUMENTATIONKEY: "foo",
+  AZURE_NH_ENDPOINT: "azendpoint",
```

Comment:
This is hard to maintain (you have to updated tests for every new env). I suggest another approach:

1. create an env.example like here: https://github.com/pagopa/io-functions-admin/blob/master/env.example
with all the variables (eventually: document them with a comment inside the env.example file)
1. make jest read the env file using this configuration: https://github.com/pagopa/io-functions-admin/blob/master/jest.config.js#L3
1. remove all references to process.env in tests files

Reply:
I have created the env file for testing, 
and added dotenv 8.2.0 as a dependency, it's ok?